docs/dataset-overviews/solar.md

```diff
@@ -96,8 +99,10 @@ The CMIP6 data can be found
 [on the ESGF under the "CMIP6" project and source ID "SOLARIS-HEPPA-3-2"](https://aims2.llnl.gov/search?project=input4MIPs&activeFacets=%7B%22mip_era%22%3A%22CMIP6%22%2C%22source_id%22%3A%22SOLARIS-HEPPA-3-2%22%7D).
 [Funke et al., GMD 2024](https://doi.org/10.5194/gmd-17-1217-2024)
 provides a detailed analysis of the changes from earlier versions.
+<!---
 \[TODO: consider whether there's a short summary we can give\]
 [TODO: get the link to slides from Bernd which dive into this]
```

Comment:
@berndfunke @st-bender I have some memory of a presentation on this. If you have the link easily to hand, could you post it here and I will add it as a further reference. (If it's not easy to get, don't worry, the paper also has lots of details.)